visualization/app/codeCharta/ui/artificialIntelligence/artificialIntelligence.component.ts

```diff
@@ -139,80 +142,80 @@ export class ArtificialIntelligenceController
 			return
 		}
 
-		const mainProgrammingLanguage = this.getMostFrequentLanguage(this.fileState.file.map)
-		this._viewModel.analyzedProgrammingLanguage = mainProgrammingLanguage
+		this.setDefaultRiskProfile()
```

Comment:
Instead of setting the default risk profile here, you can also set defaults within the viewModel-Construction.
Currently a cast is required, because you attempt to set the risk profile to a value, that should not be possible (empty object)

```ts
		riskProfile: {} as RiskProfile
```

Instead, set it straight to 
```ts
{ lowRisk: 0, moderateRisk: 0, highRisk: 0, veryHighRisk: 0 }
```

You won't need the cast then.